src/openforms/authentication/contrib/digid_eherkenning_oidc/tests/digid/test_auth_plugin_endpoint.py

```diff
@@ -21,7 +21,7 @@ def setUp(self):
 
         self.client.force_authenticate(user=self.user)
 
-    @patch("openforms.plugins.registry.GlobalConfiguration.get_solo")
+    @patch("openforms.plugins.plugin.GlobalConfiguration.get_solo")
```

Comment:
Have we changed this in all the relative files in order to do the mock where the import is done for the first time?

Reply:
yeah, it's a bit shitty because normally you don't mock things where they are defined, but this happens to work because the method is actually defined in another place.

Before this patch, the `GlobalConfiguration` model was imported in the `registry` module, but that's no longer the case, so the original mocks don't work anymore.